src/js/__tests__/__snapshots__/metadata-test.js.snap

```diff
@@ -1190,6 +1190,7 @@ Object {
   "Favorite": Array [
     "heart",
     "like",
+    "favorite",
```

Comment:
this works for now. But I think we should support "Favorite" (the object key) as a possible search. otherwise we would have to add every key in the array, which feel redundant.